docs-site/content/docs/the-app/controller.md

```diff
@@ -51,8 +51,8 @@ Controllers define Loco routes capabilities. In the example below, a controller
 ```rust
 use axum::routing::{get, post};
 Routes::new()
-    .add("/", get(hello))
-    .add("/echo", post(echo))
+    .add("/", get(hello), loco_rs::HttpMethod::GET)
```

Comment:
better to replace axum::get with loco::get so:

```
.add("/", get(hello))
```

`get(hello)` -> MethodInfo { action: GET, handler: <handler> }`